text/0748-glimmer-component-signature.md

```diff
@@ -306,10 +342,41 @@ While it's not common, occasionally components might forward `...attributes` to
 {{/if}}
 ```
 
-For such cases, components can use a union type for their `Element`. In the case of the template above, the signature would have `Element: HTMLAnchorElement | HTMLSpanElement`.
+For such cases, components can use a union type for their `Element`. In the case of the template above, the signature would have `Element: HTMLAnchorElement | HTMLSpanElement`. Correspondingly, any modifiers used with such components would need to accept any of the possible types of DOM elements declared.
 
-[`{{animated-if}}`]: https://ember-animation.github.io/ember-animated/docs/api/components/animated-if
-[`<ResponsiveImage>`]: https://github.com/kaliber5/ember-responsive-image#the-responsiveimage-component
+Similarly, a component that may use `...attributes` on an `<a>` element or may not spread them at all might write: `Element: HTMLAnchorElement | null`. In such cases, it would be legal to use any modifiers that accept an `HTMLAnchorElement`, since they wouldn't ever be invoked for the `null` scenario.
```

Comment:
It's probably worth clarifying this slightly here by adding a note that ecosystem tooling which *consumes* this information should treat it safely accordingly.